src/test/html/test-user.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>Jasmine Test HTML for users</title>
+ 
+  <link rel="shortcut icon" type="image/png" href="images/jasmine_favicon.png">
+  <link rel="stylesheet" type="text/css" href="lib/jasmine-core/jasmine.css">
+ 
+  <script type="text/javascript" src="lib/jasmine-core/jasmine.js"></script>
+  <script type="text/javascript" src="lib/jasmine-core/jasmine-html.js"></script>
+  <script type="text/javascript" src="lib/jasmine-core/boot.js"></script>
+  <script type="text/javascript" src="lib/jasmine-core/jquery-1.7.2.js"></script>
+  <script type="text/javascript" src="lib/jasmine-core/jasmine-jquery.js"></script>
+  
+  <script src="https://d3js.org/d3.v5.min.js"></script>
+  <script src="../../main/webapp/scripts/script.js"></script>
+  <script src="../../main/webapp/scripts/user-script.js"></script>
+  <script src="../javascript/test-setup.js"></script>
```

Comment:
Is this line needed? Is the test-setup.js script supposed to be run when loadfixture is called? It doesn't seem to be ran as that would lead to some kind of infinite recursion?

Reply:
It seems like based on what a fixture is https://testdrivenwebsites.com/html-fixtures-in-jasmine-using-jasmine-jquery/, it is just treated as the raw text in an html. I don't think any of these scripts should be here as I don't believe that loading the this file as a fixture will run the scripts and it may confuse someone reading the file. 

When i comment them out the tests still run.

Reply:
Yes you're right, it's unnecessary to load them in the html, I deleted them in the new commit.

Reply:
Also I updated the description of the PR in the title, besides changing load fixture method and UI changes made before, the main change is in test-user.js, added a little and all tests are working smoothly now.